src/components/WidgetContainer/index.tsx

```diff
@@ -1,7 +1,7 @@
 import { Box, Tooltip } from "@mui/material";
 import InfoIcon from "@mui/icons-material/Info";
 import React from "react";
-
+import Draggable from "react-draggable";
```

Comment:
```suggestion
import Draggable from "react-draggable";

```

Please leave an empty line after import statements.